src/content/h1/tags/model_collision_geometry/readme.md

```diff
@@ -28,11 +28,9 @@ Pathfinding spheres (blue) for a50 shown in [Sapien][] after running `debug_obje
   </figcaption>
 </figure>
 
-In Halo CE AI can figure out where to go by checking the pathfinding data on the BSP. But when you place objects in a level this of course doesn't affect the BSP. The solution that Bungie came up with for this is pathfinding spheres.
+AI can figure out where to go by checking the pathfinding data on the [BSP][scenario_structure_bsp]. However, since objects like [scenery][] and [units][unit] are not part of the BSP, Bungie implemented _pathfinding spheres_: spherical markers on objects that AI actively avoid walking into.
 
-Pathfinding spheres are spherical markers on objects that AI actively avoid walking into. This helps because it allows the author of the model to tell the AI exactly where not to go. So it won't walk into walls, or rocks or trees.
-
-As far as we know all object types can make use of pathfinding spheres. Note that pathfinding spheres outside the object's bounding sphere are ignored.
+By placing these spheres in an object's collision model, artists can tell the AI exactly where _not_. As far as we know, all object types can make use of pathfinding spheres. Note that pathfinding spheres outside the object's bounding sphere are ignored.
```

Comment:
> pathfinding spheres outside the object's bounding sphere are ignored.

Does this mean the _origin_ of the pathfinding sphere is outside the bounding sphere? Or does this mean _any part_ of the pathfinding sphere outside the bounding sphere?